lib/components/PrivacyPolicy.tsx

```
@@ -0,0 +1,462 @@
+            <p key={i}>{txt}</p>
+          ))}
+          <ul className="list-disc pl-6 space-y-2">
+            {orderedValues(tc.rights.subtext).map((txt, i) => (
+              <li key={i}>{txt}</li>
+            ))}
+          </ul>
+          {/* bottomText contains the dynamic BfDI link */}
+          {orderedValues(tc.rights.bottomText).map((txt, i) => (
+            <p
+              key={i}
+              dangerouslySetInnerHTML={{ __html: replaceLinks(txt) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)